package/src/ApiContext/api/index.ts

```diff
@@ -8,9 +8,14 @@ import { BrowserClient, Hub } from "@sentry/browser";
 import type { CryptoAddrType } from '../initialState'
 
 import { BASE_API } from './constants'
+import i18n from "../../i18n/setup"
+import { t } from "i18next"
 
 // Note: custom headers most be allowed by the preflight checks, make sure to add them to `access-control-allow-headers` corsPreflight on the server
-const headers = new Headers()
+const headers = new Headers();
+// Accept-Language header is set here for i18n. The backend will use this to set the language of the responses.
+headers.set('Accept-Language', i18n.language);
```

Comment:
- This might have unintended consequences on requests that are not made to the backend.
- The language set here might be old if the user changes it after this file has been executed. This means that the headers would still have language1 even though the user has changed it to language2, resulting in a response from the server of which its content has been translated to the old language.